extensions/docker/src/extension.ts

```diff
@@ -175,6 +176,11 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   dockerCompatibilitySetup.init().catch((err: unknown) => {
     console.error('Error while initializing docker compatibility setup', err);
   });
+  const dockerSynchronizer = new DockerContextSynchronizer(dockerContextHandler);
+  extensionContext.subscriptions.push(dockerSynchronizer);
+  dockerSynchronizer.init().catch((err: unknown) => {
```

Comment:
Original issue references macOS, should this only run on mac or is this required for Linux and Windows too?